api/types/license.go

```diff
@@ -601,4 +624,7 @@ type LicenseSpecV3 struct {
 	AnonymizationKey string `json:"anonymization_key,omitempty"`
 	// SupportsPolicy turns Teleport Policy features on or off.
 	SupportsPolicy Bool `json:"policy,omitempty"`
+
+	// entitlements define a customer’s access to a specific features
```

Comment:
should we add a comment somewhere that explains how feature support checking should be done?

eg:
```go
{ AccessMonitoring: true, Limit: 0} = full support
{ AccessMonitoring: true, Limit: LIMIT } = limited support
{ AccessMonitoring: false, limit: LIMIT } = no support 
```

to check for `full support` you'll need to check for `Entitled == true + 0 limit`

Reply:
This is documented in a sense on line 168 where we define the EntitlementInfo struct:

```
// EntitlementInfo is the state and limits of a particular entitlement
type EntitlementInfo struct {
	// Enabled indicates the feature is 'on' if true; feature is disabled if false
	Enabled Bool
	// Limit indicates the allotted amount of use when limited; if 0 use is unlimited
	Limit int32
}
```

Does this suffice or do you want me to add the explicit examples?

```
// EntitlementInfo is the state and limits of a particular entitlement; Examples:
// { FeatureX: true, Limit: 0} = unlimited access to FeatureX
// { FeatureX: true, Limit: LIMIT } = limited access to FeatureX
// { FeatureX: false, limit: LIMIT } = no access to FeatureX
type EntitlementInfo struct {
	// Enabled indicates the feature is 'on' if true; feature is disabled if false
	Enabled Bool
	// Limit indicates the allotted amount of use when limited; if 0 use is unlimited
	Limit int32
}
```

Reply:
i appreciate the explicit 👍

Reply:
Added examples in https://github.com/gravitational/teleport/pull/42394/commits/e7762b78fd40fbf99338f7311fb201b7b4fa040e